tests/systems/mad4pg_system_test.py

```diff
@@ -167,7 +170,6 @@ def test_centralised_mad4pg_on_debugging_env(self) -> None:
             checkpoint=False,
             architecture=architectures.CentralisedQValueCritic,
             trainer_fn=mad4pg.MAD4PGCentralisedTrainer,
-            shared_weights=False,
```

Comment:
I would rather explicitly set `shared_weights=True`, to make it clear that we need this in our tests